services/web/server/src/simcore_service_webserver/projects/_groups_service.py

```diff
@@ -170,6 +174,51 @@ async def delete_project_group(
     )
 
 
+async def create_confirmation_action_to_share_project(
+    app: web.Application,
+    *,
+    product_name: ProductName,
+    user_id: UserID,  # sharer
+    project_id: ProjectID,  # shared
+    sharee_email: EmailStr,  # sharee
+    # access-rights for sharing
+    read: bool,
+    write: bool,
+    delete: bool,
+) -> IDStr:
```

Comment:
maybe  calling them respectively `sharing_read` `sharing_write` and `sharing_delete` or use an object for the rights and call it `sharing_rights` would allow to not have all these comments in between.

Reply:
I find it clear by the context (ProjectShare) but i will consider in the coming PRs